docs/backup_and_restore.rst

```diff
@@ -101,89 +88,407 @@ When you are ready to begin the backup, run
+
+.. _migrate_v3:
+
+Migrating Using a V2+V3 or V3-Only Backup
+'''''''''''''''''''''''''''''''''''''''''
+
+.. note::  The instructions below assume that you are using the same *Admin Workstation*
+   that was used to manage your old instance. If you are using a new *Admin
+   Workstation* you will need to copy the directory ``~amnesia/Persistent/securedrop``
+   from the old workstation to the new workstation (using a *Transfer Device*)
+   before proceeding.
+
+#. If you have not already done so, :ref:`back up the existing installation <backing_up>`. 
+   The instructions below assume that the backup has been created 
+   and renamed ``sd-backup-old.tar.gz``.
+
+#. Move the existing *Admin Workstation* SecureDrop code out of the way, by
+   opening a Terminal via **Applications > System Tools > Terminal** and
+   running the command:
+
+   .. code:: sh
+
+      mv ~/Persistent/securedrop ~/Persistent/sd.bak
+
+#. Move the existing *Admin Workstation* SSH configuration out of the way via
+   the Terminal, using the commands:
+
+   .. code:: sh
+
+       mv ~/.ssh/config ~/.ssh/config.bak
+       mv ~/.ssh/known_hosts ~/.ssh/known_hosts.bak
+
+#. Re-clone the SecureDrop repository to the *Admin Workstation* using the following 
+   Terminal commands:
+
+   .. code:: sh
+
+      cd ~/Persistent
+      git clone https://github.com/freedomofpress/securedrop
+
+#. Verify that the current release tag was signed with the release signing key:
+
+   .. code:: sh
+
+      cd ~/Persistent/securedrop/
+      git fetch --tags
+      git tag -v 1.7.1
+
+   The output should include the following two lines:
+
+   .. code:: sh
+
+      gpg:                using RSA key 22245C81E3BAEB4138B36061310F561200F4AD77
+      gpg: Good signature from "SecureDrop Release Signing Key"
+
+
+   .. important::
+      If you do not see the message above, signature verification has failed
+      and you should **not** proceed with the installation. If this happens,
+      please contact us at securedrop@freedom.press.
+
+
+   Verify that each character of the fingerprint matches what is on the
+   screen of your workstation. If it does, you can check out the new release:
+
+   .. code:: sh
+
+      git checkout 1.7.1
+
+   .. important::
+      If you see the warning ``refname '1.7.1' is ambiguous`` in the
+      output, we recommend that you contact us immediately at
+      securedrop@freedom.press 
+      (`GPG encrypted <https://securedrop.org/sites/default/files/fpf-email.asc>`__).
+
+#. Copy the old instance's configuration files and backup from 
+   ``~/Persistent/sd.bak`` into ``~/Persistent/securedrop`` using the following 
+   Terminal commands:
+
+   .. code:: sh
+
+      cd ~/Persistent/securedrop
+      export SD_OLD="~/Persistent/sd.bak/install_files/ansible-base"
+      export SD_NEW="~/Persistent/securedrop/install_files/ansible-base"
+      cp $SD_OLD/group_vars/all/site-specific $SD_NEW/group_vars/all/site-specific
```

Comment:
In case folks type those commands by hand (and to make them easier to read), I would suggest  omitting the destination filename in all `cp` operations where it is not modified. So:

```bash
cp $SD_OLD/group_vars/all/site-specific $SD_NEW/group_vars/all/
cp $SD_OLD/tor_v3_keys.json $SD_NEW/
cp $SD_OLD/sd-backup-old.tar.gz $SD_NEW/
cp $SD_OLD/SecureDrop.asc $SD_NEW/
cp $SD_OLD/ossec.asc $SD_NEW/
cp $SD_OLD/journalist.asc $SD_NEW/
```

This is done further below for some of the commands.

Reply:
gotcha, fixed in  7f2b421